.husky/pre-commit

```diff
@@ -1 +1 @@
-pnpm exec lint-staged && pnpm run types
+pnpm lint && pnpm run types
```

Comment:
lint-staged is not the same as lint command. It allows to run linter only for changed files so let's keep this for pre commit.